core/src/main/java/org/jdbi/v3/core/mapper/BuiltInMapperFactory.java

```diff
@@ -1,216 +0,0 @@
-/*
- * Licensed under the Apache License, Version 2.0 (the "License");
- * you may not use this file except in compliance with the License.
- * You may obtain a copy of the License at
- *
- * http://www.apache.org/licenses/LICENSE-2.0
- *
- * Unless required by applicable law or agreed to in writing, software
- * distributed under the License is distributed on an "AS IS" BASIS,
- * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
- * See the License for the specific language governing permissions and
- * limitations under the License.
- */
-package org.jdbi.v3.core.mapper;
-
-import java.lang.reflect.Type;
-import java.math.BigDecimal;
-import java.net.InetAddress;
-import java.net.URI;
-import java.net.URISyntaxException;
-import java.net.URL;
-import java.net.UnknownHostException;
-import java.sql.ResultSet;
-import java.sql.SQLException;
-import java.sql.Time;
-import java.sql.Timestamp;
-import java.time.Instant;
-import java.time.LocalDate;
-import java.time.LocalDateTime;
-import java.time.LocalTime;
-import java.time.OffsetDateTime;
-import java.time.ZoneId;
-import java.time.ZonedDateTime;
-import java.util.HashMap;
-import java.util.Map;
-import java.util.Optional;
-import java.util.OptionalDouble;
-import java.util.OptionalInt;
-import java.util.OptionalLong;
-import java.util.UUID;
-import java.util.function.Function;
-import java.util.function.Supplier;
-import org.jdbi.v3.core.config.ConfigRegistry;
-import org.jdbi.v3.core.statement.StatementContext;
-
-import static org.jdbi.v3.core.generic.GenericTypes.getErasedType;
-
-/**
- * Column mapper factory which knows how to map JDBC-recognized types, along with some other well-known types
- * from the JDK.
- */
-public class BuiltInMapperFactory implements ColumnMapperFactory {
```

Comment:
This is removing what is technically public API.
I doubt anyone uses it - it's installed by default.  But perhaps we should leave it as deprecated and preserve its functionality by moving the list of installed factories from `ColumnMappers` to here?

Reply:
That also leaves this class as the start of your `OotbPlugin`

Reply:
How do you mean exactly, move the list there?

Reply:
You could preserve existing behavior by delegating to a list of all the new mapper factories:

```java
private static final List<ColumnMapperFactory> FACTORIES = new ArrayList<>();

static {
        FACTORIES.add(new EnumMapperFactory());
        FACTORIES.add(new OptionalMapperFactory());
        FACTORIES.add(new PrimitiveMapperFactory());
        FACTORIES.add(new BoxedMapperFactory());
        FACTORIES.add(new EssentialsMapperFactory());
        FACTORIES.add(new InternetMapperFactory());
        FACTORIES.add(new SqlTimeMapperFactory());
        FACTORIES.add(new JavaTimeMapperFactory());
        FACTORIES.add(new OptionalMapperFactory());}
}

public Optional<ColumnMapper<?>> build(Type type, ConfigRegistry config) {
  return FACTORIES.stream()
    .flatMap(factory -> JdbiStreams.toStream(factory.build(type, config)))
    .findFirst();
}
```